modules/tensor_mechanics/include/utils/RankTwoScalarTools.h

```diff
@@ -385,6 +393,92 @@ radialStress(const RankTwoTensorTempl<T> & stress,
   return radial_stress;
 }
 
+/* The radial stress is calculated as
+ * radial_stress = radial^T_i * \sigma_{ij} * radial_j
+ * where normal is the position vector of the current point that is normal to
+ * the user-defined axis of rotation;
+ * @param center The center point of the spherical system
+ * @param curr_point The point corresponding to the stress (pass in & _q_point[_qp])
+ * @param direction The direction vector in which the scalar stress value is calculated.
+ */
+template <typename T>
+T
+radialStress(const RankTwoTensorTempl<T> & stress,
+             const Point & center,
+             const Point & curr_point,
+             Point & direction)
+{
+  Point radial = curr_point - center;
+  radial /= radial.norm();
+
+  // Compute the scalar stress component in the radial direction
+  T radial_stress = 0.0;
+  for (unsigned int i = 0; i < 3; ++i)
+    for (unsigned int j = 0; j < 3; ++j)
+      radial_stress += radial(j) * stress(j, i) * radial(i);
```

Comment:
wouldn't this just be `radial * (stress * radial)`? Why the manual looping over components?

Reply:
Great point. I think that is correct. I changed them all.

Reply:
Thanks! I'm working on the Mandel notation classes and we need to get rid of as much component access as we can to make it feasible to template on these classes....